src/execution_impl.rs

```diff
@@ -23,26 +23,30 @@ use crate::middlewares::gas_calibration::{get_gas_calibration_result, GasCalibra
 /// The return of the function executed as string and the remaining gas for the rest of the execution.
 pub(crate) fn exec(
     limit: u64,
-    instance: Option<Instance>,
+    instance_and_store: Option<(Instance, Store)>,
     mut module: impl MassaModule,
     function: &str,
     param: &[u8],
-) -> Result<(Response, Instance)> {
-    let instance = match instance {
-        Some(instance) => instance,
-        None => create_instance(limit, &module)?,
+) -> Result<(Response, Instance, Store)> {
+    let (instance, mut store) = match instance_and_store {
+        Some((instance, store)) => (instance, store),
+        None => create_instance(limit, &mut module)?,
     };
-    module.init_with_instance(&instance)?;
+    // module.init_with_instance(&instance, &store)?;
```

Comment:
Already done in `create_instance` let's remove this to avoid misleading anyone

Reply:
removed